pkg/netstore/netstore_test.go

```diff
@@ -225,7 +225,7 @@ type retrievalMock struct {
 	addr      swarm.Address
 }
 
-func (r *retrievalMock) RetrieveChunk(ctx context.Context, addr swarm.Address, orig bool) (chunk swarm.Chunk, err error) {
+func (r *retrievalMock) RetrieveChunk(ctx context.Context, addr, sourceAddr swarm.Address) (chunk swarm.Chunk, err error) {
```

Comment:
I think that would benefit is having a better naming and vocabulary, for instance it's clear that `addr` is of the type Address, but more context would be helpful.

Reply:
I added more documentation on the retrieval interface in the retrieval protocol. You're commenting on a mock. I think it would make more sense to document the interface properly, since it is effectively the one we should relate to, no?

Reply:
yep, makes sense!